src/bundle/Resources/public/js/OnlineEditor/buttons/ez-btn-styleslist.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import AlloyEditor from 'alloyeditor';
+import EzButtonDropdown from './ez-btn-dropdown';
+
+export default class EzButtonStylesList extends AlloyEditor.ButtonStylesList {
+
+    render() {
+        var e;
```

Comment:
let/const instead of var

Reply:
@GrabowskiM 

this method is taken directly out of AlloyEditor as I mentioned here: https://github.com/ezsystems/ezplatform-richtext/pull/115#discussion_r382578977